frontend/components/LineCountList.tsx

```diff
@@ -19,7 +19,7 @@ export default function LineCountList() {
   }, []);
   return (
     <div>
-      <div className={"messagelist-header"}> Current Line Count : </div>
```

Comment:
It's usual styling to write every word in a heading capitalized, besides fillers like for etc, so i'd do the same here, or go with everything small letters

Reply:
Fixed and  comitted with doom emacs